packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2108,10 +2108,13 @@ class TextSelectionGestureDetectorBuilder {
     switch (defaultTargetPlatform) {
       case TargetPlatform.android:
       case TargetPlatform.fuchsia:
+        // On mobile platforms the selection is set on tap up.
+        editableText.hideToolbar(false);
```

Comment:
I tried this on Android with a physical mouse and the handle remains visible:

https://user-images.githubusercontent.com/389558/236953666-f3d8c797-6a5d-49ea-8825-d5cbc5a59512.mp4